src/Laravel/ApiPlatformProvider.php

```diff
@@ -936,51 +940,6 @@ public function register(): void
             );
         });
 
-        if ($this->app['config']->get('api-platform.graphql.enabled')) {
```

Comment:
it's not good to read configuration inside the register method as it may change when `boot` is called